package.json

```diff
@@ -24,7 +24,9 @@
     "@aws-cdk/aws-logs": "1.109.0",
     "@aws-cdk/aws-s3-deployment": "1.109.0",
     "@aws-cdk/core": "1.109.0",
-    "archiver": "^5.3.1",
+    "@typescript-eslint/eslint-plugin": "^5.23.0",
+    "archiver": "^5.3.0",
+    "eslint": "^8.15.0",
```

Comment:
I think these changes to `package.json` and `yarn.lock` are no longer needed so should be removed.